src/subcommand/init.rs

```diff
@@ -43,7 +44,11 @@ impl Init {
         match self.hook {
             Hook::none => (),
             Hook::prompt => writeln!(handle, "{}", config.hook.prompt).unwrap(),
-            Hook::pwd => writeln!(handle, "{}", config.hook.pwd).unwrap(),
+            Hook::pwd => {
+                if let Some(pwd_hook) = config.hook.pwd {
```

Comment:
If the user has set `--hook pwd` on an unsupported shell, we should explicitly `bail!` and let them know, eg. `PWD hooks are currently unsupported on this shell`.

Reply:
That would require this function returning a `Result`. Are you fine with that?

Reply:
Absolutely!